### ISSUE TITLE
Please add radio for demo

### ISSUE THREAD

**Issue by tomByrer** [dbohdan/classless-css#41]
Seems radio is missing from demo.

Also I'm thinking a test for wrapping checkbox/radio with `<label>` instead of using `for` helps.

**Comment by dbohdan** [comment]
Good idea. I am going to add both. I am not sure when and if I will regenerate the screenshots for existing stylesheets, but the elements will be there for new ones.
